src/core/textrenderer/qgstextrenderer.h

```diff
@@ -264,6 +299,15 @@ class CORE_EXPORT QgsTextRenderer
      */
     static constexpr double FONT_WORKAROUND_SCALE = 10;
 
+    // to match QTextEngine handling of superscript/subscript font sizes
```

Comment:
Is that comment a copy/paste artifact?

Reply:
No, it relates to the constant value below -- SUPERSCRIPT_SUBSCRIPT_FONT_SIZE_SCALING_FACTOR